src/pages/EnablePayments/IdologyQuestions.js

```diff
@@ -54,149 +51,119 @@ const defaultProps = {
     walletAdditionalDetails: {},
 };
 
-class IdologyQuestions extends React.Component {
-    constructor(props) {
-        super(props);
-        this.submitAnswers = this.submitAnswers.bind(this);
-
-        this.state = {
-            /** Current question index to display. */
-            questionNumber: 0,
+function IdologyQuestions({questions, walletAdditionalDetails, idNumber}) {
+    const formRef = useRef();
+    const {translate} = useLocalize();
 
-            /** Should we hide the "Skip question" answer? Yes if the user already skipped MAX_SKIP questions. */
-            hideSkip: false,
+    const [currentQuestionIndex, setCurrentQuestionIndex] = useState(0);
+    const [shouldHideSkipAnswer, setShouldHideSkipAnswer] = useState(false);
+    const [userAnswers, setUserAnswers] = useState([]);
+    const [error, setError] = useState('');
 
-            /** Answers from the user */
-            answers: [],
-
-            /** Any error message */
-            errorMessage: '',
-        };
-    }
+    const currentQuestion = questions[currentQuestionIndex] || {};
+    const possibleAnswers = _.filter(
+        _.map(currentQuestion.answer, (answer) => {
+            if (shouldHideSkipAnswer && answer === SKIP_QUESTION_TEXT) {
+                return;
+            }
 
-    /**
-     * Put question answer in the state.
-     * @param {Number} questionIndex
-     * @param {String} answer
-     */
-    chooseAnswer(questionIndex, answer) {
-        this.setState((prevState) => {
-            const answers = prevState.answers;
-            const question = this.props.questions[questionIndex];
-            answers[questionIndex] = {question: question.type, answer};
             return {
-                answers,
-                errorMessage: '',
+                label: answer,
+                value: answer,
             };
-        });
-    }
+        }),
+    );
+    const errorMessage = error;
+    // const errorMessage = ErrorUtils.getLatestErrorMessage(walletAdditionalDetails) || error;
```

Comment:
can we remove this commented line or add a comment why it is commented?

Reply:
my mistake, that should be uncommented and line above should be deleted